HUGODEHSIGN/my-otaku-opinions
# 11 hello world deployment

## Purpose

Finished the server.js and connection.js files

Fixes #11 

## Changes

- path/to/file
  - config/connection.js
  - server.js
  - 

## Steps to Reproduce or Test

- clone the branch
- use npm i in terminal to install packages
- confirm that server starts
